spec/controllers/splash_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it 'is valid with valid attributes' do
+    user = build(:user)
+    expect(user).to be_valid
+  end
+
+  it 'is not valid without a name' do
+    user = build(:user, name: nil)
+    expect(user).not_to be_valid
+  end
+
+  it 'has many categories' do
+    user = User.reflect_on_association(:categories)
+    expect(user.macro).to eq(:has_many)
+  end
+
+  it 'has many payments' do
+    user = User.reflect_on_association(:payments)
+    expect(user.macro).to eq(:has_many)
+  end
+
+  it 'is confirmable' do
+    user = create(:user)
+    expect(user.confirmed?).to be true
+  end
+end
```

Comment:
@Dachrono I kindly suggest you check this file as it does not belong to the splash screen. These tests already exist in your app so you can easily remove this duplicate file.

Reply:
Tky for your feedback I really appreciate it, Solved